apps/api/src/front.js

```diff
@@ -21,6 +23,21 @@ const initContextState = ctx => {
     ctx.state.internalError = false;
     ctx.state.request = {};
     ctx.state.success = false;
+    ctx.state.stats = {};
+};
+
+const getStats = async ctx => {
```

Comment:
I think it would be better to forward the db client as function argument and send back the stats? I'm not a fan of this context transmittion the mutation, because we are not middleware.